### ISSUE TITLE
UI Scaled Incorrectly on MacOS

### ISSUE THREAD

**Issue by wongandj** [sirjuddington/SLADE#1706]
### SLADE Version

3.2.5

### OS

macOS

### Editor

Resource editor

### Steps to Reproduce and Observed Behavior

1. Install SLADE on MacOS.
2. Open the program and look at it.
3. Attempt to set the preferences and notice the confirmation button doesn't fit on the screen.
4. Attempt to convert graphics and notice the confirmation button doesn't fit on the screen.
5. Notice that nothing fits on the screen, even in fullscreen.

### Expected Behavior

I expected the UI to be at a normal scale, like previous versions were.

### Screenshots

![Full screen; Notice broken zoom widget](https://github.com/user-attachments/assets/18c077bd-eef7-4bd3-ad43-b77c66e161f5)
![Prefernces; Notice inaccessible confirm button](https://github.com/user-attachments/assets/87981b9f-282c-480a-909a-6c8621950bf9)

**Comment by Pedro-Beirao** [comment]
This happens because your monitor has hiDPI enabled.
If you disable it using apps like BetterDisplay, then it will render normally.

What I find weird is that if slade is started with hiDPI disabled, but then I enable hiDPI, it will keep rendering fine.

I dont know how hard it would be to fix this issue on hiDPI for Slade/wxwidgets, but its something that I will look into in the future.

**Comment by 107zxz** [comment]
Would it be worth making a PR to disable hiDPI on Mac builds? I could put that together

**Comment by 107zxz** [comment]
Building the master branch from source fixed this for me, but releases are still bugged.

**Comment by sirjuddington** [comment]
That's good to know, I think the linux DPI fixes I made probably fixed the issue in macOS too. @Pedro-Beirao are you still around and able to make macOS releases?

**Comment by Pedro-Beirao** [comment]
> That's good to know, I think the linux DPI fixes I made probably fixed the issue in macOS too. @Pedro-Beirao are you still around and able to make macOS releases?

Yeah for sure. Ive been busy with dsda-doom, but have been planning to look back into Slade.

I can make a new macOS release, but would you instead be interested in using Github Actions for building every commit and release? I can make a PR for Windows and macOS

**Comment by sirjuddington** [comment]
Oh yeah that would be a great help if you can get it going via GitHub Actions when you get the time, thanks :)

**Comment by sirjuddington** [comment]
Closing this as it's fixed since 3.2.5
